modules/core/test/v2/unit/wallet.ts

```diff
@@ -422,9 +434,238 @@ describe('V2 Wallet:', function () {
             forwarderVersion: 1,
           },
         });
-      await ethWallet.createAddress({ chain: 0, forwarderVersion: 1 }).should.be.rejectedWith('address validation failure: expected 0x32a226cda14e352a47bf4b1658648d8037736f80 but got 0x8c13cd0bb198858f628d5631ba4b2293fc08df49');
+       await ethWallet.createAddress({ chain: 0, forwarderVersion: 1 }).should.be.rejectedWith('address validation failure: expected 0x32a226cda14e352a47bf4b1658648d8037736f80 but got 0x8c13cd0bb198858f628d5631ba4b2293fc08df49');
     });
-  });
+
+    it('verify address when invalid baseAddress is passed', async function () {
+      nock(bgUrl)
```

Comment:
please store the return value and ensure the nock is consumed for all these nocks.

Reply:
this has not been resolved @AbhayShekhawat94

Reply:
Done